packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -23,22 +23,40 @@ export interface Props extends Themeable {
   prefix?: string;
   suffix?: string;
   decimals?: number;
+  displayMode: 'basic' | 'lcd' | 'gradient';
```

Comment:
Looks like this could be an enum?

Reply:
Yes, this is temporary thing , needs a variable

Reply:
I mean enum